src/kcm/kcm.cpp

```
@@ -215,6 +224,10 @@ void MaterialDecorationKCM::save()
     m_settings->setDragFromButtonsEnabled(m_ui->kcfg_DragFromButtonsEnabled->isChecked());
 
     m_settings->save();
+
+    Material::InternalSettingsList exceptions(m_ui->exceptions->exceptions());
```

Clang (14.0.6)

[warning] variable 'exceptions' is not initialized

(cppcoreguidelines-init-variables)